charts/karpenter/templates/flowschema.yaml

```diff
@@ -0,0 +1,76 @@
+{{ if .Values.flowSchema.leaderElection.create }}
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta3
```

Comment:
Should we create an issue to track this graduating to stable in v1.29?

Reply:
Yep, good callout

Reply:
Added #5060